### ISSUE TITLE
Add "Sense" like capability per circuit

### ISSUE THREAD

**Issue by JamesAndrews352** [MarkBryanMilligan/LanternPowerMonitor#8]
While most loads are obvious, for example, the water heater is the only load on the water heater breaker. A kitchen may have a dishwasher, microwave, and garbage disposal on the same circuit. I'm not proposing some fancy artificial Intelligence, just the ability to turn on an appliance and have the system recognize that specific load, then it would be broken out under that circuit so you know how much each appliance uses. Maybe more complicated than I think.

**Comment by MarkBryanMilligan** [comment]
Yeah, I could see the use for an intermediate mode, not really what sense does.  So like you just manually figure out what each device uses by watching the instantaneous power while you turn it on and off.  Oh, ok, my microwave uses 1300 watts.  Now I'll configure a watch item for this breaker, if the power jumps between 1200 and 1400 up or down, well then that's the microwave turning on or off.  Basically manually train it for each thing.  Then you add in some extra logic in the summary aggregation that watches for those changes and puts the power in a subgroup when the state changes.  I bet that's about a week project.

**Comment by kitlaan** [comment]
I ran across the [NILMTK](https://github.com/nilmtk/nilmtk) while googling around, which seems to be research around this area.  It's all voodoo to me so maybe someone else can determine what's required to utilize it.

At the very least, a way to store/export data in HDF5 format may be useful to help the community and/or allow external integrations?  (Aside, the data storage requirement is probably high, so storing on a SD card seems unwise? USB storage? NAS?)
